lib/utils/note-utils.test.js

```diff
@@ -19,7 +16,7 @@ describe('noteTitleAndPreview', () => {
 
   it('should return default values if note content is empty', () => {
     const result = noteTitleAndPreview(note);
-    expect(result).toEqual(defaults);
+    expect(result).toEqual({"preview": "", "title": "New Note..."});
```

Comment:
at some point we need to get our auto-formatter working and then this will be `{preview: '', title: 'New Note...' }` automatically

Reply:
I never turned prettier on after my system upgrade.  Agreed that we need to do this on commit.